db/migrate/20230124181134_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
+      t.references :author, null: false, foreign_key: true
+
```

Comment:
- [ ] Kindly make sure that the reference foreign key is tied to a table. Here you're referencing a table called "authors" which doesn't exist. (please do this to all tables that have a reference to the "users" table)

- [HINT] You can use the following syntax to do so.
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```